### PR TITLE
Adapt form error UI to design

### PR DESCRIPTION
**Pull request by bolom**
Adjusts the UI for form errors according to https://invis.io/Z7OD63XAWKF#/467087061_New_Sample_-_Erros

### CODE REVIEW

**Comment by leandroradusky** [comment]
I was performing a smoke test, and it seems to work OK. **But**, there is an odd change on the date pickers:

Before (`main` branch): 

image

After:

image

Same is happening with all the date pickers.

**Comment by bolom** [comment]
@leandroradusky  I've updated the css

**Review by leandroradusky — APPROVED** [review]
I went through the code & tested it locally and it looks good!

File: app/helpers/cdx_form_helper.rb [hunk 1]
```diff
@@ -79,53 +79,80 @@ def form_actions(&block)
...
 
-    Rails.logger.info "Unhandled form errors in #{@object.model_name}: #{unhandled_errors}"
-    Raven.capture_message("Unhandled form errors", extra: {
-      form: @object.model_name,
+    Rails.logger.info message
+    Raven.capture_message("Unhandled form errors",
+      form: object_name,
       errors: unhandled_errors,
-    })
+    )
```
Comment (ysbaddaden) [inline app/helpers/cdx_form_helper.rb:156]: I don't know how the new Sentry gem is working, but I just verified the legacy Raven source code, and we must specify the `extra:` key (see `Raven::Event.new`).

```ruby
    Raven.capture_message("Unhandled form errors", extra: {
      form: object_name,
      errors: unhandled_errors,
    })
```

File: app/helpers/cdx_form_helper.rb [hunk 2]
```diff
@@ -79,53 +79,80 @@ def form_actions(&block)
...
+    else
+      errors_to_show.clear
+    end
+  end
+
   def log_unhandled_errors
     unhandled_errors = error_messages_to_show.reject { |k, _| k == :base }
     return if unhandled_errors.empty?
 
+    message = "Unhandled form errors in #{object_name}: #{unhandled_errors}"
```
Comment (ysbaddaden) [inline app/helpers/cdx_form_helper.rb:147]: Where does the `object_name` variable come from?

**Review by ysbaddaden — CHANGES_REQUESTED** [review]
Thanks @bolom but I'm kinda lost: why bring 28de0127e55d8ead250ea8b8ce5fd8e12bc9fcac from #1689 which we closed as too complex in favor to #1706? The only commit we want in is f0b6086ebea6970c99bce700e10ab681af2a560b from #1682, rebased from `main` and without the above commit from #1689.

Related slack discussion: 
https://manas.slack.com/archives/C034BR64J/p1684274918617989?thread_ts=1684268802.208069&cid=C034BR64J

I just tried a `git cherry-pick f0b6086ebea6970c99bce700e10ab681af2a560b` and the commit applies cleanly on top of main. Once `CdxFormHelper#has_errors?` is fixed, the design is in.

**Comment by bolom** [comment]
@ysbaddaden voila, pushed my commit with your requests.

**Review by ysbaddaden — APPROVED** [review]
Perfect!

**Pull request merged**